joint_teapot/utils/joj3.py

```diff
@@ -179,7 +179,7 @@ def generate_failed_table(
 
 
 def generate_title_and_comment(
-    score_file_path: str, action_link: str, run_number: str
+    score_file_path: str, action_link: str, run_number: str, exercise_name: str
```

Comment:
This `exercise_name` should also be added to other 2 joj3 commands. And it should use the comment of metadata stage (line 58) as a fallback if exercise name is Unknown. Just for a smoother migration.

Reply:
How to have access to the data written in metadata stage? I don't have access to the scoreboard, so I'm not quite sure

Reply:
Changed the code to let the user set the exercise_name at metadata procedure.